boards/opentitan/Makefile

```diff
@@ -6,9 +6,11 @@ PLATFORM=opentitan
 include ../Makefile.common
 
 qemu: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).elf
+	$(call check_defined, OPENTITAN_BOOT_ROM)
```

Comment:
Is there an "expected correct" default value we could give to `OPENTITAN_BOOT_ROM` so calling `make qemu` from the `opentitan/` directory just works? I'm not really sure why it needs to be a variable in the first place.

Reply:
You need to download the file and we need to know where it is. The CI should do this for you, but for users they probably want to specify where it is.

Reply:
I think this is a case where if you want to step outside of what the CI does for you, you need to know this for yourself. The OT + qemu stuff is too alpha to try to do more right now.